src/typing_extensions.py

```diff
@@ -2657,7 +2657,9 @@ def __init__(self, getitem):
             self.__doc__ = _UNPACK_DOC
 
     class _UnpackAlias(typing._GenericAlias, _root=True):
-        __class__ = typing.TypeVar
+        if sys.version_info[:2] < (3, 11):
```

Comment:
```suggestion
        if sys.version_info < (3, 11):
```
Minor but more consistent with how we usually write version checks in typed code